assets/chezmoi.io/docs/reference/command-line-flags/common.md

```diff
@@ -54,11 +57,6 @@ Also perform command on all parent directories of *target*.
 
 Recurse into subdirectories, `true` by default.
 
-## `--source-path`
```

Comment:
This should not be moved from `common.md` as it does not apply to *all* commands, only a large subset.

Reply:
Same.